frontend/appflowy_flutter/lib/plugins/document/document_page.dart

```diff
@@ -209,18 +207,19 @@ class _DocumentPageState extends State<DocumentPage>
                 }
               }
 
-              await editorState!.dropImages(
-                data.dropTarget!,
-                imageFiles,
-                widget.view.id,
-                isLocalMode,
-              );
-              await editorState!.dropFiles(
-                data.dropTarget!,
-                otherFiles,
-                widget.view.id,
-                isLocalMode,
-              );
+              // TODO(Lucas): add back the image and file drop logic
+              // await editorState!.dropImages(
+              //   data.dropTarget!,
+              //   imageFiles,
+              //   widget.view.id,
+              //   isLocalMode,
+              // );
+              // await editorState!.dropFiles(
+              //   data.dropTarget!,
+              //   otherFiles,
+              //   widget.view.id,
+              //   isLocalMode,
+              // );
```

Comment:
Do you want me to fix this?

Reply:
I can do that later. I want to include the drag feature to see if it's okay to integrate.